doc/source/user_guide/ug_case_settings.rst

```diff
@@ -39,6 +39,8 @@ to support the CICE model.
    "ICE_DRVOPT", " ", "unused", "icepack"
    "ICE_IOTYPE", " ", "unused", "none"
    "ICE_CLEANBUILD", "true,false", "automatically clean before building", "true"
+   "ICE_CPPDEFS", " ", "user defined cppdefs for build", "null"
```

Comment:
```suggestion
   "ICE_CPPDEFS", " ", "user defined preprocessor macros for build", "null"
```
This seems to be the standard terminology: 
https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html#Preprocessor-Options

Also, maybe it is worth it to mention that the `-D` flag must be used, i.e. `-DCICE_MACRO` and not just `CICE_MACRO` ?